### PR TITLE
Fix RBAC bugs

### PR DESCRIPTION
**Pull request by marounkoussaifi**

### CODE REVIEW

File: src/test/java/org/ow2/proactive/catalog/service/CatalogObjectGrantServiceTest.java [hunk 1]
```diff
@@ -263,31 +263,6 @@ public void testDeleteAllCatalogObjectsGrantsAssignedToABucket() {
         verify(catalogObjectGrantRepository, times(1)).delete(dbUserObjectGrants);
     }
 
-    @Test
-    public void testDeleteAllCatalogObjectGrants() {
```
Comment (fviale) [inline src/test/java/org/ow2/proactive/catalog/service/CatalogObjectGrantServiceTest.java:267]: why did you remove this test ?

**Comment by fviale** [comment]
Retest this please

**Review by tao-xinxiu — APPROVED** [review]

File: src/test/java/org/ow2/proactive/catalog/service/CatalogObjectGrantServiceTest.java (hunk 1 above)
Comment (marounkoussaifi) [reply src/test/java/org/ow2/proactive/catalog/service/CatalogObjectGrantServiceTest.java:267]: I removed the function that it was testing.

Comment (marounkoussaifi) [reply src/test/java/org/ow2/proactive/catalog/service/CatalogObjectGrantServiceTest.java:267]: The function was deleting grants when the object was deleted. Since I changed this mechanism by adding the "cascade" the functions is no longer needed. I removed it then and remove its test.

Comment (fviale) [reply src/test/java/org/ow2/proactive/catalog/service/CatalogObjectGrantServiceTest.java:267]: ok

**Review by fviale — APPROVED** [review]
